### ISSUE TITLE
508-defect-3 [COGNITION]: COVID Vaccine - Apply button label should be more descriptive 

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#17113]
# [508-defect-3](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-3)

## Feedback framework

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

## Point of Contact

**VFS Point of Contact:** _Trevor_

## User Story or Problem Statement

As a user who might have a cognitive disability or short-term memory loss, I want the "Apply" button to offer more context as what I am applying for, without having to scroll further back up the page to re-read the heading.

## Details

Our "Apply" button does not offer context what users are applying for without being able to see or review the page heading. On mobile devices, that heading requires users to scroll back to the top of the screen, then back down to submit their form. I have mocked up a more descriptive text in the button and placed it in the Chrome Dev Tools mobile device sim. Screen shot attached below.

Discussed with @DanielleThierryUSDSVA at 5:30 pm ET on December 8th:
> Button should say "**Sign up to stay informed**" and we will have an ARIA label for screen readers (shown in the code snippet below).

## Acceptance Criteria

- [ ] input[type="submit"] visual text updated to say "Sign up to stay informed"
- [ ] input[type="submit"] ARIA label text should say "Sign up to stay informed about the COVID-19 vaccine"
- [ ] Zero axe browser plugin violations
- [ ] Button fits in the mobile-width browser viewport without horizontal scrolling

## Code Snippet
```diff
<button
+ aria-label="Sign up to stay informed about the COVID-19 vaccine"
  type="submit"
  class="usa-button"
>
- Apply
+ Sign up to stay informed
</button>
```

## WCAG or Vendor Guidance (optional)

* [Link Purpose (Link Only): Understanding SC 2.4.9](https://www.w3.org/TR/UNDERSTANDING-WCAG20/navigation-mechanisms-link.html)

## Screenshots or Trace Logs

![Screen Shot 2020-12-08 at 4 58 43 PM](https://user-images.githubusercontent.com/934879/101551585-d953bd00-3976-11eb-8de6-d016cb833c94.png)

**Comment by 1Copenut** [comment]
This has been tested and verified in Staging with multiple screen readers. Moving to close.
